src/styles/index.css

```diff
@@ -0,0 +1,3 @@
+@import url(./blocks/card.css);
+@import url(./blocks/stars.css);
+@import url(./reset.css);
```

Comment:
./reset.css has to be first
@import url(./reset.css);
@import url(./blocks/card.css);
@import url(./blocks/stars.css);